app/helpers/user_helper.rb

```diff
@@ -0,0 +1,19 @@
+helpers do
+
+  def session_logged_in?
+    !session[:current_user_id].blank?
+  end
+
+  def session_current_user
+    User.find(session[:current_user_id])
+  end
+
+  def current_users_photos
+    session_current_user.photos
+  end
+
+  def current_users_tags
```

Comment:
making a method called `current_users_tags` when you could just call `current_user.tags` seems unnecessary.